bin/morgue.js

```diff
@@ -5951,6 +5952,17 @@ function coronerLogin(argv, config, cb) {
     });
   }
 
+  const loginCb = (username, password) => {
+    coroner.login(username, password, function(err) {
+      loginComplete(coroner, argv, err, cb);
+    })
+  };
+
+  if (process.env.MORGUE_USERNAME && process.env.MORGUE_PASSWORD) {
```

Comment:
does this do what we want on windows?

Reply:
It should, but because all of our services are using SSO I don't have a convenient way to test it.  The Windows invocation of the command is different--for powershell you have to do something like `$Env:VAR=bla` 3 times as separate commands I think, but the process.env thing is a Node feature that's not OS-specific.